frappe/public/js/frappe/ui/tree.js

```diff
@@ -154,7 +155,7 @@ frappe.ui.Tree = class {
 			]);
 		} else {
 			frappe.run_serially([
-				() => {return this.get_all_nodes(value, is_root);},
+				() => {return this.get_all_nodes(value, is_root, lab);},
```

Comment:
> __[ESLint]__ Statement inside of curly braces should be on next line. ([view](https://sider.review/gh/repos/1864194/pulls/10967#issue-4134649))
>
> <details>
>
> | Rule | Severity | Category | Recommended |
> | ---- | -------- | -------- | ----------- |
> | `brace-style` | `error` | Stylistic Issues | No |
>
> References:
> - <https://eslint.org/docs/rules/brace-style>
>
> You can [close this issue](https://sider.review/gh/repos/1864194/pulls/10967#issue-4134649) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

The brace formatting on this line is identical to that in "develop." Happy to change it if you prefer, but this commit did not create the condition Sider is objecting to. Please just let me know how you'd like me to proceed.

Reply:
```suggestion
				() => this.get_all_nodes(value, is_root, lab),
```
@gwhitney it'll be great if you can fix this for the other lines as well!

Reply:
OK, I think I made all of the analogous changes, and it seems to work, so updated the PR. Let me know if the new version looks OK (of course I will look at what Sider says in a minute.